Library/Formula/apr.rb

```diff
@@ -23,8 +25,12 @@ def install
     # The internal libtool throws an enormous strop if we don't do...
     ENV.deparallelize
 
+    args = %W[--prefix=#{libexec}]
+
+    args << "--build=i386-apple-darwin" if build.universal?
```

Comment:
Yeah, it does an ` uname -m ` to check for the arch at compile time, so it usually automatically pulls it in.

```
checking build system type... x86_64-apple-darwin14.1.0
checking host system type... x86_64-apple-darwin14.1.0
checking target system type... x86_64-apple-darwin14.1.0
Configuring APR library
Platform: x86_64-apple-darwin14.1.0
```